### PR TITLE
fix(vm-docs): Add section for changing vulnerability status

### PR DESCRIPTION
**Pull request by cpoggioli-newrelic**
## Give us some context

* This PR adds documentaiton for the new NRVM feature that will allow users to change the vulnerability status of an entity

Screenshot 2024-05-06 at 3 40 54 PM
Screenshot 2024-05-06 at 3 41 04 PM
Screenshot 2024-05-06 at 3 41 13 PM

### CODE REVIEW

File: src/content/docs/vulnerability-management/changing-vulnerability-status.mdx [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+import vmAffectedBulkDetails from 'images/security_screenshot-affected-bulk-details.webp'
+
+import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'
+
+Many organizations have an overwhelming number of vulnerabilities detected within their system. But, not all of these vulnerabilities are high-risk, have remediations, or are run on systems of high operational importance or external exposure. NRVM allows you to ignore vulnerabilities on a per-entity basis, this allows for dashboards to only contain vulnerabilities that match your risk threshold and ensure your teams focus on impactful remediations. 
+
+## Requirements 
+
+- Vulnerability Management requires 'write' permissions for the [vulnerabilities capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#security) in order for a user to change the status of a vulnerability.
```
Comment (akristen) [inline src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:39]: Is there a version requirement for agents or for vulnerability management that customers need to be on?

Comment (akristen) [reply src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:39]: Does VM with this feature happen out of the box on install?

File: src/content/docs/vulnerability-management/changing-vulnerability-status.mdx [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+## Ignoring a vulnerability
+
+Not all vulnerabilities detected by Vulnerability Management may require immediate attention. Using Vulnerability Management, you can manually ignore vulnerabilities on a per-entity basis. Once you set the status of an instance to ignored, we’ll exclude this instance from the summary tiles and vulnerability list by default.
```
Comment (akristen) [inline src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:43]: Why might a developer or security person want to ignore a vulnerability? How can we help them make a risk as sessment? For example, in alerts we have this idea of "alert fatigue" -- getting too many notifications about certain behaviors. Can we provide examples of when a vulnerability could be deprioritized, even if it's more high level and not specific?

File: src/content/docs/vulnerability-management/changing-vulnerability-status.mdx [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+
+import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'
+
+Many organizations have an overwhelming number of vulnerabilities detected within their system. But, not all of these vulnerabilities are high-risk, have remediations, or are run on systems of high operational importance or external exposure. NRVM allows you to ignore vulnerabilities on a per-entity basis, this allows for dashboards to only contain vulnerabilities that match your risk threshold and ensure your teams focus on impactful remediations. 
+
+## Requirements 
+
+- Vulnerability Management requires 'write' permissions for the [vulnerabilities capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#security) in order for a user to change the status of a vulnerability.
+
+## Ignoring a vulnerability
```
[the remark this answers is not in the document]
Comment (cpoggioli-newrelic) [reply src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:41]: Good callout, I'll check with our PM/Director.

File: src/content/docs/vulnerability-management/changing-vulnerability-status.mdx (hunk 1 above)
Comment (cpoggioli-newrelic) [reply src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:39]: There is a general doc for NRVM agent versions. (https://docs.newrelic.com/docs/vulnerability-management/integrations/intro/#apm-agents) 
This feature happens out of the box on install but requires users have the 'write' permission. Custom roles will not have this permission by default.

File: src/content/docs/vulnerability-management/changing-vulnerability-status.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (cpoggioli-newrelic) [reply src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:41]: We are messaging this in ask-security-legal-compliance to get an answer.

File: src/content/docs/vulnerability-management/changing-vulnerability-status.mdx [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+        <img
+            title="Vulnerabilities tab with ignored column added"
+            alt="An image showing the all entities vulnerabilities tab with an ignore column added"
+            src={vmAffectedBulkVulnList}
+        />
+    </Step>
+    <Step>
+        #### Change the filter
+
+        Click the filter **Status = Affected** and change it to **Ignored**. Now only entities with the vulnerability marked as ignored will be visible.
```
Comment (akristen) [inline src/content/docs/vulnerability-management/changing-vulnerability-status.mdx:256]: What's the difference between this step and the step that adds the ignore column? When might someone choose one method over the other?

**Comment by akristen** [comment]
https://github.com/newrelic/docs-website/pull/17229 Final PR here that used this branch as its base branch! Closing this one

**Pull request closed without merging**